frontend/src/components/dashboard/home/ActiveVisitorsCard.vue

```diff
@@ -58,28 +70,64 @@
     export default {
         name: 'ActiveVisitorsCard',
         props: {
-            activeUsersCount: {
-                type: Number,
-                default: 250
-            },
-            pageViewsCount: {
-                type: Number,
-                default: 31
+            data: {
+                type: Array,
+                required: true,
             },
-            topPages: {
+            activityChartData: {
                 type: Array,
-                default: () =>  {
-                    return ['link_1/juhy/kkk', 'link_2/juhy/klk', 'link_3/juk/jjj'];
+                required: true,
+            },
+            isFetching: {
+                type: Boolean,
+                required: true,
+            },
+        },
+        data: () => ({
+            lineWidth: 5,
+            radius: 16,
+            padding: 4,
+            gradient: ['#3C57DE', '#1BC3DA'],
+            gradientDirection: 'left',
+        }),
+        computed: {
+            chartData() {
```

Comment:
This computed prop probably redundant, since you can use `activityChartData` in template directly, unless you going to add more logic here later